src/main/resources/fabric.mod.json

```diff
@@ -21,7 +21,7 @@
 	],
 	"depends": {
 		"fabricloader": ">=0.15.11",
-		"minecraft": "~1.20.6",
+		"minecraft": ["1.20.6","1.21"],
```

Comment:
(I doubt this would actually be compatible with 1.20.6 but it'll be fun laughing at whoever tries to run it against that)